<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>datas</class>
 <widget class="QWidget" name="datas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>757</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>15</y>
        <width>189</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Сохранение данных&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>273</width>
        <height>116</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QCheckBox" name="checkBox">
         <property name="text">
          <string>Данные идентификации</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="checkBox_2">
         <property name="text">
          <string>Данные электромагнитных процессов</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="checkBox_3">
         <property name="text">
          <string>Данные тепловых процессов</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QCheckBox" name="checkBox_4">
         <property name="text">
          <string>Данные вентиляционной сети</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QWidget" name="widget_2" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>55</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="enterDannie">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>33</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Ввести данные</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveDannie">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>33</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>33</height>
         </size>
        </property>
        <property name="text">
         <string>Сохранить данные</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignVCenter">
       <widget class="QPushButton" name="deleteDannie">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>33</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>33</height>
         </size>
        </property>
        <property name="text">
         <string>Удалить данные</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" rowspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Список сохраненных двигателей&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableView" name="tableView"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Источник данных&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton">
        <property name="text">
         <string>Осциллограф+наблюдатель скорости (без датчика скорости)</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_4">
        <property name="text">
         <string>Осциллограф+наблюдатель момента (с датчиком скорости)</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_3">
        <property name="text">
         <string>Внутренний источник данных</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_2">
        <property name="text">
         <string>Чтение данных из файла  для идентификации параметров схемы замещения</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QPushButton" name="pushButton_5">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Обзор</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_5">
        <property name="text">
         <string>Чтение данных из файла  для наблюдателя скорости</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QPushButton" name="pushButton_7">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Обзор</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>saveDannie</tabstop>
  <tabstop>deleteDannie</tabstop>
  <tabstop>tableView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
